github/repos.go

```diff
@@ -972,3 +973,52 @@ func (s *RepositoriesService) RemoveAdminEnforcement(ctx context.Context, owner,
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// Topics represents a repository topics to list or replace.
```

Comment:
Maybe `represents a collection of repository topics` ? I think that reads a bit better.